### ISSUE TITLE
Web -  Messaged non existing account, then you get an error if you try to login into that account

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#21483]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Log in with an existing account
2. Search for an email that you know doesn't have an account and start a chat
3. In another session, try to login with the email you started a chat with
4. You get an error: Cannot get account details, please try again or contact [concierge@expensify.com](mailto:concierge@expensify.com)

## Expected Result:
No error message  "Cannot get account details, please try again or contact [concierge@expensify.com](mailto:concierge@expensify.com)"

## Actual Result:
Error  message displayed "Cannot get account details, please try again or contact [concierge@expensify.com](mailto:concierge@expensify.com)"

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** n/a

**Reproducible in staging?:** n/a

**Reproducible in production?:** n/a

**If this was caught during regression testing, add the test name, ID and link from TestRail:**

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

![image (77)](https://github.com/Expensify/App/assets/93399543/96d42c2f-43b0-41c9-a6b2-d38816b67547)

**Expensify/Expensify Issue URL:**

**Issue reported by:** @aldo-expensify 

**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1686949198475289

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by kadiealexander** [comment]
@aldo-expensify I think this might be fixed (tested on chrome with never-used email kadielizzy+21483@gmail.com), could you please confirm?

![image](https://github.com/Expensify/App/assets/59587260/e619d8f2-2303-4077-a4fa-d50922d006c9)

![image](https://github.com/Expensify/App/assets/59587260/2bb2e4e0-6505-4ba6-8bae-e88936c3db23)

**Comment by kadiealexander** [comment]
Ah actually, this seems to only happen with expensify.com and expensifail.com emails:

![image](https://github.com/Expensify/App/assets/59587260/0659a5a1-8724-436f-b642-4531c1be47b0)

![image](https://github.com/Expensify/App/assets/59587260/1cd946cd-e06b-41db-8d68-31d51113d46b)

I have a hunch that this is related to Domain Control.

**Comment by kadiealexander** [comment]
@madmax330 this has only been reproduced on expensify.com and expensifail.com accounts, but I have a feeling it might be related to Domain Controlled accounts in general (In OldDot, domain managed emails need their account to be created by a Domain Admin). I added the `Internal` label based on this, but if you feel this could be external after a look please feel free to switch the labels.

**Comment by madmax330** [comment]
Yeah agreed this is internal. I think it's because when we create a chat, we create the account in a closed state. So when you go to log in it doesn't work

**Comment by kadiealexander** [comment]
Not overdue.

**Comment by kadiealexander** [comment]
@madmax330 any thoughts on this one?

**Comment by madmax330** [comment]
I haven't had time to look into this, been super swamped lately, but will look into it today.

**Comment by kadiealexander** [comment]
@madmax330 any thoughts so far?

**Comment by madmax330** [comment]
I was super swamped last week, but will prioritize it this week

**Comment by kadiealexander** [comment]
@madmax330 any updates?

**Comment by madmax330** [comment]
Going to look into it this week

**Comment by kadiealexander** [comment]
Any updates?

**Comment by madmax330** [comment]
Focusing on other higher priority issues during offshore, will get back to this later

**Comment by kadiealexander** [comment]
Not overdue

**Comment by kadiealexander** [comment]
@madmax330 any updates here?

**Comment by kadiealexander** [comment]
@madmax330 bump!

**Comment by madmax330** [comment]
Okay so I figured out the problem. This will happen for all validated domains. 
I think the solution here is to block being able to chat to people on validated domains who don't yet have an account. Since those accounts are controlled by the domain admin normally right?

**Comment by madmax330** [comment]
Posted here: https://expensify.slack.com/archives/C03TQ48KC/p1698942981544569

**Comment by madmax330** [comment]
We just need to update the error message as this is expected behavior since the account is on domain control, the user cannot just create an account without the admin.

**Comment by madmax330** [comment]
Same, working on EOY issues

**Comment by kadiealexander** [comment]
Closing as a niche bug that doesn't really fit in a wave and only impacts internal employees.
